job-service-db/src/main/resources/db/migration/functions/public/R__cancelJob.sql

```diff
@@ -20,11 +20,13 @@
  *  Description:
  *  Cancels the specified job.
  */
+DROP FUNCTION IF EXISTS cancel_job(character varying,character varying);
+
 CREATE OR REPLACE FUNCTION cancel_job(
     in_partition_id VARCHAR(40),
     in_job_id VARCHAR(48)
 )
-RETURNS VOID
+RETURNS BOOLEAN
```

Comment:
Do we need this change since the function only ever returns true?

Reply:
We don't - if there is an issue with cancelling a job, an exception will be raised.
Otherwise, the cancellation is a success and we increment the counter in cancel_jobs